### PR TITLE
RTC config instead of remote.html

### PR DESCRIPTION
**Pull request by alexflorisca**
AMP people changes the way ads work with custom targeting. This PR reflects that.

### CODE REVIEW

File: server/lib/transforms/html/insert-ad.js [hunk 1]
```diff
@@ -2,20 +2,10 @@
...
-	<div class="ad-container">
-		<amp-ad
-			width="300"
-			height="250"
-			type="doubleclick"
-			data-slot="/5887/ft.com/distributed.content/amp.ft"
-			rtc-config='{"urls":["https://ads-api.ft.com/v1/content/${uuid}?amp=true","https://ads-api.ft.com/v1/user?amp=true", "https://cdn.krxd.net/userdata/v2/amp/HINzStCn?segments_key=ksg&kuid_key=kuid"]}'>
-		</amp-ad>
-	</div>`;
+const adMarkup = uuid => `<div class="ad-container"><amp-ad	width="300"	height="250" type="doubleclick"	data-slot="/5887/ft.com/distributed.content/amp.ft" rtc-config='{"urls":["https://ads-api.ft.com/v1/content/${uuid}?amp=true","https://ads-api.ft.com/v1/user?amp=true", "https://cdn.krxd.net/userdata/v2/amp/HINzStCn?segments_key=ksg&kuid_key=kuid"]}'></amp-ad></div>`;
```
Comment (quarterto) [inline server/lib/transforms/html/insert-ad.js]: was there a particular reason to remove the whitespace here?

**Comment by quarterto** [comment]
AMP RTC documentation for posterity: https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/rtc-documentation.md

File: package-lock.json [hunk 2]
```diff
@@ -1,7264 +0,0 @@
-{
```
Comment (quarterto) [inline package-lock.json]: can you reinstate this please?

File: server/lib/transforms/html/insert-ad.js (hunk 1 above)
Comment (alexflorisca) [reply server/lib/transforms/html/insert-ad.js]: @quarterto Yep, the dom inserts text nodes between actual dom nodes if there are spaces and the tests started failing. Fun FFUN

**Pull request merged**